app/org/sagebionetworks/bridge/models/accounts/StudyParticipant.java

```diff
@@ -122,14 +147,33 @@ public String getId() {
         private boolean notifyByEmail;
         private Set<String> dataGroups = ImmutableSet.of();
         private String healthCode;
-        private Map<String,String> attributes = Maps.newHashMap();
-        private Map<String,List<UserConsentHistory>> consentHistories = Maps.newHashMap();
-        private Set<Roles> roles = Sets.newHashSet();
+        private Map<String,String> attributes = ImmutableMap.of();
+        private Map<String,List<UserConsentHistory>> consentHistories = ImmutableMap.of();
+        private Set<Roles> roles = ImmutableSet.of();
```

Comment:
If you want to make these immutable, you should also have the setters or build() make immutable copies.